src/utils/ustring_utils.cc

```diff
@@ -53,19 +52,39 @@ namespace Astroid {
       trim_right(str);
   }
 
-  ustring UstringUtils::random_alphanumeric (int length) {
+  ustring UstringUtils::random_string_from_charset (const ustring chars, const int length) {
     ustring str;
+    uint32_t r;
 
-    const string _chars = "abcdefghijklmnopqrstuvwxyz1234567890";
-    random_device rd;
-    mt19937 g(rd());
-
-    for (int i = 0; i < length; i++)
-      str += _chars[g() % _chars.size()];
+    for (int i = 0; i < length; i++) {
+      r = arc4random_uniform(chars.length());   // This is recommended over constructions
```

Comment:
Seems to be missing an include here?

Reply:
Hm, compiles for me. Do you get an error? Usually it's in `stdlib.h`.